libqrexec/libqrexec-utils.h

```diff
@@ -53,8 +54,21 @@ int flush_client_data(int fd, struct buffer *buffer);
 int write_stdin(int fd, const char *data, int len, struct buffer *buffer);
 int fork_and_flush_stdin(int fd, struct buffer *buffer);
 
-void do_fork_exec(const char *cmdline, int *pid, int *stdin_fd, int *stdout_fd,
-        int *stderr_fd);
+/**
+ * @param cmdline Null-terminated command to execute.
+ * @param argument The argument to pass, or NULL for no argument.
```

Comment:
This description seems inaccurate - `argument` is a (IMO misleadingly named) argv array for new process, so both program path (overriding `cmdline`) and its arguments.

Reply:
I agree.